tests/cpp/accuracy/test_accuracy.cpp

```diff
@@ -122,8 +128,43 @@ TEST_P(ModelParameterizedTest, AccuracyTest)
             ASSERT_EQ(prediction_buffer.str(), modelData.testData[i].reference[0]); // Check top-1 only
         }
     }
+    else if (modelData.type == "SegmentationModel") {
+        auto model = SegmentationModel::create_model(DATA_DIR + "/" + modelPath);
+
+        for (size_t i = 0; i < modelData.testData.size(); i++) {
+            auto imagePath = DATA_DIR + "/" + IMAGE_PATH + "/" + modelData.testData[i].image;
+
+            cv::Mat image = cv::imread(imagePath);
+            if (!image.data) {
+                throw std::runtime_error{"Failed to read the image"};
+            }
+            cv::Mat predicted_mask[] = {model->infer(image)->resultImage};
```

Comment:
Based on the recent changes in the validation of the Python part. Don't you need to take min/max from the `predicted_mask` prior to defining the ranges for the histogram?

Reply:
Ok. I see that this is true according to the postprocessing defined in the segmentation model_type.